crm/modules/member/install.inc.php

```diff
@@ -28,7 +28,7 @@
 function member_install($old_revision = 0) {
     global $db_connect;
     if ($old_revision == 1) {
-        error_log('The database version is too old to upgrade to this release of ' . title(). '.  Please upgrade one release at a time.');
+        error_log('The database version is too old to upgrade to this release of ' . title() . '. Please upgrade one release at a time.');
```

Comment:
Woo! that's some confusing stuff right there.  Which dot is concatenate and which dot is the actual period in the sentence!?  Good catch here.